sopsinstall.sh

```diff
@@ -62,13 +62,12 @@ adjust_binary() {
 # wrap all destructive operations into a function
 # to prevent curl|bash network truncation and disaster
 execute() {
-  TMPDIR=$(mktmpdir)
+  TMPDIR=$(mktemp -d)
```

Comment:
the culprit. it still doesn't clean up after itself though, but at least its not doing something actually bad.